### DISCUSSION TITLE
How to manually update via OTA?

### DISCUSSION THREAD

**Discussion by PepperPhil** [ESPresense/ESPresense#290] [Q&A]
Sorry, I searched here a lot and could not really find an answer for ESP32 dummies, like me. 
How do you update w/o clearing the config and w/o connecting the ESP32 to the computer?

**Comment by DTTerastar** [comment]
It is a bunch of command line python at the moment.  I don't know of a really easy way, yet.  The newest web install has an option NOT to clear your settings so its the easiest way to fix a failed upgrade.
